### ISSUE TITLE
Generate validation set for finetuned classes

### ISSUE THREAD

**Issue by aZij1Cam** [DFKI-Interactive-Machine-Learning/py-faster-rcnn-ft#5]
We want to evaluate if the fine tuned network performs better than the VGG_CNN_M_1024 caffemodel.

We want to get sets  that can be input into customToolbox.py functions get_val_images show_val_images instead of minival2014.
